Insecure script reference causes ui-ace to fail to load when served via HTTPS

```
Mixed Content: The page at 'https://[REDACTED]/' was loaded over HTTPS, but requested an insecure script 'http://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/ace.js'. This request has been blocked; the content must be served over HTTPS.
angular.min.js:118 Error: ui-ace need ace to work... (o rly?)
    at ui-ace.min.js:7:155
    at Object.invoke (angular.min.js:41:456)
    at angular.min.js:53:3
    at q (angular.min.js:7:355)
    at Object.<anonymous> (angular.min.js:52:476)
    at Object.invoke (angular.min.js:41:456)
    at Object.$get (angular.min.js:39:142)
    at Object.invoke (angular.min.js:41:456)
    at angular.min.js:43:265
    at Object.d [as get] (angular.min.js:40:432)
```

--- a/index.html
+++ b/index.html
@@ -31,7 +31,7 @@
     <script type="text/javascript" src="./build/js/prettify/run_prettify.js"></script>
     
     <!-- previous version of ace caused import conflict -->
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/ace.js" type="text/javascript" charset="utf-8"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/ace.js" type="text/javascript" charset="utf-8"></script>
     
     <script type="text/javascript" src="build/js/dld4e-connections.js"></script>
     <script type="text/javascript" src="build/js/dld4e-draw.js"></script>

